### PR TITLE
Read placeholder size from graph

### PR DESCRIPTION
**Pull request by nanowireUK**
## Purpose
Fixed issue #39 - the Custom Vision exported model has a changed input size. According to other samples it is now expected that the input size is read from the graph rather than hardcoded at 227.
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
Train and export new custom vision model. Previously would not work, now does.
```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by nanowireUK** [comment]
Used the pattern described in the [official sample](https://docs.microsoft.com/en-us/azure/cognitive-services/custom-vision-service/export-model-python#crop-the-center-for-the-specific-input-size-for-the-model)

File: modules/ImageClassifierService/app/predict.py [hunk 1]
```diff
@@ -25,7 +24,7 @@
 
 def initialize():
     print('Loading model...',end=''),
-    with tf.gfile.FastGFile(filename, 'rb') as f:
```
Comment (emmanuel-bv) [inline modules/ImageClassifierService/app/predict.py]: Thanks a lot for your contribution. Why are you replacing FastGFile by GFile? The official sample seems to be using FastGFile

**Review by emmanuel-bv — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution. Why are you replacing FastGFile by GFile? The official sample seems to be using FastGFile

File: modules/ImageClassifierService/app/predict.py (hunk 1 above)
Comment (nanowireUK) [reply modules/ImageClassifierService/app/predict.py]: That was actually an unrelated change I put in to resolve a deprecation warning I was getting. I have amended my PR to remove this change.

**Review by nanowireUK — COMMENTED** [review]
made change

**Review by emmanuel-bv — APPROVED** [review]
Thanks a lot for your contribution!

**Pull request merged**